lib/bucky/core/test_core/test_case_loader.rb

```diff
@@ -148,6 +148,12 @@ def refine_case_with_option(test_cond, suite)
               suite[:cases].delete_if { |c| c[:labels].nil? }
               suite[:cases].delete_if { |c| !(test_cond[:label].sort - [c[:labels]].flatten.sort).empty? }
             end
+            # Exclude by label
+            if test_cond.key? :xlabel
+              # Delete test case that have specify label
+              suite[:cases].delete_if { |c| c[:labels].nil? }
```

Comment:
```suggestion
```

Reply:
I think test cases that have no labels should be run.

Reply:
I think when using the xl (exclude label) option to fitter the test cases, it should depend on all the cases got the label.

Reply:
Fixed!